index.html

```diff
@@ -302,13 +280,7 @@ <h3 id="streaming-document-examples-valid">Valid Examples</h3>
               </pre>
               <pre class="example" title="Valid streaming document with @context, plain @type, @id, and other properties">
                 {
-                  "@context": {
-                    "Person": "http://schema.org/Person",
-                    "name": "http://schema.org/name",
-                    "url": "http://schema.org/url",
-                    "image": "http://schema.org/image"
-                  },
-                  "@id": "https://www.rubensworks.net/#me",
```

Comment:
I think this @id was removed by mistake.

Reply:
Oops, indeed, that was a mistake...